alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -341,7 +341,15 @@ public static function handleImportFinished(BatchJob $dbBatchJob, kImportJobData
 				kJobsManager::addCapturaThumbJob($entryThumbnail->getPartnerId(), $entryThumbnail->getEntryId(), $entryThumbnail->getId(), $srcSyncKey, $flavorAsset->getId(), $srcAssetType, $thumbParamsOutput);
 			}
 		}
-		
+
+		$replacingEntry = $dbBatchJob->getEntry();
+		if ($replacingEntry &&
+			$replacingEntry->getIsTemporary() &&
+			kReplacementHelper::shouldSyncFlavorInfo($flavorAsset, $flavorAsset->getEntryId()))
+		{
+			kReplacementHelper::copyReadyReplacingEntryAssetToReplacedEntry($flavorAsset);
```

Comment:
copyReadyReplacingEntryAssetToReplacedEntry can throw propel exception, check if needed to catch and make them more accurate and relevant